deps/rabbit/src/rabbit_amqp_management.erl

```diff
@@ -160,6 +159,10 @@ handle_http_req(HttpMethod = <<"PUT">>,
                     %% Must have been created in the meantime. Loop around again.
                     handle_http_req(HttpMethod, PathSegments, Query, ReqPayload,
                                     Vhost, User, ConnPid, {PermCache2, TopicPermCache});
+                {protocol_error, queue_limit_exceeded, Reason, ReasonArgs} ->
```

Comment:
This is not a protocol error per se, just a general `error` that has to do with the state (and specific configuration) of the cluster.

I'd use `{error, queue_limit_exceeded, Reason, ReasonArgs}` as @ansd suggested below.